aiida/cmdline/commands/cmd_restapi.py

```diff
@@ -12,60 +12,54 @@
 Main advantage of doing this by means of a verdi command is that different
 profiles can be selected at hook-up (-p flag).
 """
-import os
 
 import click
 
-import aiida.restapi
 from aiida.cmdline.commands.cmd_verdi import verdi
 from aiida.cmdline.params.options import HOSTNAME, PORT
-
-CONFIG_DIR = os.path.join(os.path.split(os.path.abspath(aiida.restapi.__file__))[0], 'common')
+from aiida.restapi.common import config
 
 
 @verdi.command('restapi')
-@HOSTNAME(default='127.0.0.1')
-@PORT(default=5000)
+@HOSTNAME(default=config.CLI_DEFAULTS['HOST_NAME'])
+@PORT(default=config.CLI_DEFAULTS['PORT'])
 @click.option(
     '-c',
     '--config-dir',
     type=click.Path(exists=True),
-    default=CONFIG_DIR,
-    help='the path of the configuration directory'
+    default=config.CLI_DEFAULTS['CONFIG_DIR'],
+    help='Path to the configuration directory'
 )
-@click.option('--debug', 'debug', is_flag=True, default=False, help='run app in debug mode')
+@click.option('--debug', 'debug', is_flag=True, default=config.APP_CONFIG['DEBUG'], help='Enable debugging')
```

Comment:
Should this default maybe also come from `CLI_DEFAULTS`?

Reply:
Well, the variable was already in `APP_CONFIG`, so I did not want to change this.

Reply:
Sure, but it seems to be a different variable then. It is not actually a default for the cli, you are just co-opting it. If you want to keep in in `APP_CONFIG` then I think you should simply add a separate one to `CLI_DEFAULT`. Or should these two values always be the same anyway?

Reply:
> Or should these two values always be the same anyway?

There should only be one default value.

Perhaps the issue is rather that `CLI_DEFAULT` is not the best name.
I chose it to keep the default values together that I needed to add for the cli + for run_api, so they aren't really just meant for the cli.
I could also call them `RUN_API_DEFAULTS` or something but I'm not sure that's really better.
Let me know.

Reply:
I'm not familiar enough with the REST API to make that call. Fine to keep it as is